.github/workflows/test_deploy.yml

```diff
@@ -69,6 +69,7 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install coveralls setuptools wheel
         python -m pip install .[all]
+        python setup.py --version
```

Comment:
@Panaetius hope u dont mind that i've added this for the docs building job.

Reply:
No I think that can be useful